src/custom/strings.js

```diff
@@ -1,10 +1,11 @@
 import React from "react";
 
 import CameraRearTwoToneIcon from "@material-ui/icons/CameraRearTwoTone";
-import welcome_logo_svg from "assets/images/logo.svg";
-import welcome_globe_png from "assets/images/globe.png";
-import welcome_chart_png from "assets/images/chart.png";
-import welcome_croud_png from "assets/images/croud.png";
+import welcome_logo_svg from "custom/assets/images/logo.svg";
```

Comment:
there are two or three places with assets, and when I'ver merged the changes done by Plastic Patrol, they got mixed up in some how. Surerly the cusome images should go in a custom folder, but not sure it is the right one. I need to investigate where the assets should go...